.travis.yml

```diff
@@ -18,6 +18,7 @@ install:
 - python bootstrap.py
 - bin/buildout buildout:test-eggs=$TEST_EGGS annotate
 - bin/buildout buildout:test-eggs=$TEST_EGGS
+- pip install requests
```

Comment:
The bin/code-analysis script is failing.

```
Flake8..........................[ FAILURE ] in 0.772s
/home/travis/build/collective/collective.behavior.amp/src/collective/behavior/amp/validators.py:8:1: I003 isort expected 1 blank line in imports, found 0
/home/travis/build/collective/collective.behavior.amp/src/collective/behavior/amp/validators.py:9:1: I003 isort expected 1 blank line in imports, found 0
/home/travis/build/collective/collective.behavior.amp/src/collective/behavior/amp/validators.py:10:1: E302 expected 2 blank lines, found 1
/home/travis/build/collective/collective.behavior.amp/src/collective/behavior/amp/validators.py:50:9: T001 print statement found.
/home/travis/build/collective/collective.behavior.amp/src/collective/behavior/amp/validators.py:52:9: T001 print statement found.
/home/travis/build/collective/collective.behavior.amp/src/collective/behavior/amp/validators.py:54:13: T001 print statement found.
/home/travis/build/collective/collective.behavior.amp/src/collective/behavior/amp/validators.py:54:20: W292 no newline at end of file
The command "bin/code-analysis" exited with 1 in 0.786s.
```

See: https://travis-ci.org/collective/collective.behavior.amp/jobs/251056533